Emby.Dlna/Main/DlnaEntryPoint.cs

```diff
@@ -122,6 +125,13 @@ public sealed class DlnaEntryPoint : IServerEntryPoint, IRunBeforeStartup
                 httpClientFactory,
                 config);
             Current = this;
+
+            _netConfig = config.GetConfiguration<NetworkConfiguration>("network");
+            _disabled = appHost.ListenWithHttps && _netConfig.RequireHttps;
+            if (_disabled)
```

Comment:
Why disable it? DLNA uses separate sockets. Is there an API component that is required for DLNA?

Reply:
DLNA relies on http. The device communicates over ssdp first to receive the http connection information of ther server, but then uses http requests to control content. (Or in 10.7's case https, hence this PR)

If http is disabled (RequiredHttps is true, and the licence has loaded correctly), then won't http be disabled in which case, the controller isn't listening on http, meaning the client won't get a response over http. 

In this instance, dlna might as well stop itself running as it;'s using resources for no purpose.

Reply:
A possible feature to be proposed after this fix, would be having something like "Allow DLNA over HTTP" on a third setting (alongside "Enable HTTPS" and "Require HTTPS") thus making HTTPS actually **mandatory** only for the Web UI and any calls to any of its APIs.

I understand this would require some work on many other areas, then right now just disabling DLNA when HTTPS is required will suffice.